src/Tribe/Views/V2/View.php

```diff
@@ -2145,4 +2149,58 @@ protected function get_container_data() {
 
 		return $data;
 	}
+
+	/**
+	 * Filters Whether the Recent Past Events Should Show for a specific View.
+	 *
+	 * @since TBD
+	 *
+	 * @return mixed|void
+	 */
+	protected function should_show_recent_past_events_view() {
+		$show = $this->context->get( 'show_recent_past', true );
```

Comment:
I just want to add a note for anyone reviewing this that might look for a `show_recent_past` location added to the Context: there is none and it's fine.

This will allow us, in the future, to add an option and expose a UI to toggle if the Recent Past Events view should show or not and add the corresponding Context location then.